### PR TITLE
Create a pack for ThreatExchange

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Description
Create a pack for ThreatExchange and add readme file 

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ThreatExchange/Integrations/ThreatExchange/integration-ThreatExchange.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ script:
                 Body: method == 'POST' ? encodeToURLQuery(parameters).substring(1) : ''
             },
             params.insecure,
-            params.useproxy
+            params.useproxyf
```
Comment (ronykoz) [inline Packs/ThreatExchange/Integrations/ThreatExchange/integration-ThreatExchange.yml:65]: proxyf?

File: Packs/ThreatExchange/README.md [hunk 2]
```diff
@@ -0,0 +1,402 @@
+## Overview
+---
+
+Receive threat intelligence about applications, IP addresses, URLs and hashes, a service by Facebook
+This integration was integrated and tested with version xx of ThreatExchange
```
Comment (ronykoz) [inline Packs/ThreatExchange/README.md:5]: update the version

**Comment by ronykoz** [comment]
@kirbles19 could you please go over the readme?

File: Packs/ThreatExchange/README.md (hunk 2 above)
Comment (barchen1) [reply Packs/ThreatExchange/README.md:5]: I dont think there is a version for it cause its a service by facebook

Comment (ronykoz) [reply Packs/ThreatExchange/README.md:5]: they should have a version.. if not let remove this line

**Comment by kirbles19** [comment]
@barchen1 Is there anything to review here, or was the integration just put into package format?

**Comment by barchen1** [comment]
@kirbles19 yes, The Readme file.

https://github.com/demisto/content/blob/1e98d06731a6cdd26f2a73aeea53b03cb27393be/Packs/ThreatExchange/README.md

**Comment by kirbles19** [comment]
@barchen1 I think the changes should be made to the YML and then regenerate the README after.

**Comment by ronykoz** [comment]
@kirbles19 this is not a new entity, it's an old integration which we converted to the new format and added readme to, if you think that we need to fix the yml as well that's ok, but i believe you already checked it out at some point.

**Comment by kirbles19** [comment]
> @kirbles19 this is not a new entity, it's an old integration which we converted to the new format and added readme to, if you think that we need to fix the yml as well that's ok, but i believe you already checked it out at some point.

Even I can be edited.
I think it's worth taking a look at the YML and regenerating README.

**Comment by ronykoz** [comment]
@kirbles19 sounds good to me

**Comment by kirbles19** [comment]
@barchen1 @ronykoz Done. Can regenerate README now.

**Pull request merged**